build/apple/contentScope.js

```diff
@@ -3994,22 +4038,19 @@
                   throw new DOMException('Permission denied')
               }
 
-              const successHandler = args[0];
-              args[0] = function (position) {
-                  signalPermissionStatus(Permission.Geolocation, Status.Active);
-                  successHandler?.(position);
-              };
-              const id = DDGReflect.apply(target, thisArg, args);
-              watchedPositions.add(id);
-              return id
+              const messageId = Math.random().toString();
```

Comment:
Use a uuid instead of a random number: `const messageId =crypto.randomUUID()`

Reply:
Nice, thank you! Was looking for an existing alternative, didn't find one.